scripts/apply-phpunit-patches.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+# All commands below must no fail
+set -e
+
+# Be in the root dir
+cd "$(dirname $0)/../"
+
+find tests/ -type f -print0 | xargs -0 sed -i 's/function setUpBeforeClass(): void/function setUpBeforeClass()/g';
+find tests/ -type f -print0 | xargs -0 sed -i 's/function setUp(): void/function setUp()/g';
+find tests/ -type f -print0 | xargs -0 sed -i 's/function tearDown(): void/function tearDown()/g';
+
+sed -i 's/endTest(\\PHPUnit\\Framework\\Test \$test, float \$time): void/endTest(\\PHPUnit_Framework_Test \$test, \$time)/g' tests/functional/ReportSauceLabsStatusListener.php;
+sed -i 's/: void/ /g' tests/functional/ReportSauceLabsStatusListener.php;
+# Drop the listener from the config file
+sed -i '/<listeners>/,+2d' phpunit.xml.dist;
+sed -i 's/function runBare(): void/function runBare()/g' tests/functional/WebDriverTestCase.php;
+
+# Return back to original dir
+cd - > /dev/null
```

Comment:
@OndraM @williamdes would it be okay with you two if I used this script for https://github.com/algolia/algoliasearch-client-php/pull/644 as well?

Reply:
Yes you can copy it :)

Reply:
👍